Functions.js

```diff
@@ -0,0 +1,20 @@
+/* В терминале оплаты сохранено ваше имя, напишите функцию для определения 
+имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)*/
+function getAccess() {
+  let currenUserName = prompt("Ваше имя, сэр","");
+  return (currenUserName == "Артем")? ("Привет" + " " + currenUserName) : 'Нет такого имени';
```

Comment:
1) Please always use `===` instead of `==` in JS.
2) Forgotten space before `?`.
3) You can use interpolation of strings for getting more readable code. Read more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals